src/content/en/updates/2017/10/devtools-release-notes.md

```diff
@@ -53,14 +53,15 @@ of VS Code and DevTools in action, side-by-side.
 </div>
 
 Multi-client remote debugging was the [number 1 most-popular DevTools issue
-on Crbug][Crbug], and number 3 across the entire Chromium project.
+on crbug.com][Crbug], and number 3 across the entire Chromium project.
 Multi-client support also  opens up quite a few interesting opportunities
 for integrating other tools with DevTools, or using those tools in new
 ways. For example:
 
 [Crbug]: https://crbug.com/129539
 
-* Two WebSocket protocol clients, such as two Puppeteer sessions, can now
+* An open DevTools window plus a connected protocol client. This includes Selenium/Chromedriver, Chrome debugging within VS Code or WebStorm, and WebSocket clients like Puppeteer. 
+* Two separate WebSocket protocol clients (eg. [Puppeteer](https://github.com/GoogleChrome/puppeteer) or [chrome-remote-interface](https://github.com/cyrus-and/chrome-remote-interface)), can now
```

Comment:
Writer nitpick: Can you change this to:

`Two separate WebSocket protocol clients, such as [Puppeteer](...) or [chrome-remote-interface](...), can now...`

Using something like "such as" rather than "e.g." comes from our style guide: https://developers.google.com/style/abbreviations#dont-use